docs/index.md

```diff
@@ -2,32 +2,43 @@
 
 <p style="font-size: larger; font-weight: bold;">Transform your research with vast amounts of computing!</p>
 
-The OSG Virtual School 2021 will prepare you to use high throughput computing (HTC) systems&nbsp;&mdash;
-locally or using the OSG&nbsp;&mdash;
+The OSG Virtual School 2021 will prepare you to use high throughput computing (HTC)&nbsp;&mdash;
```

Comment:
I’m mostly fine with this change, except (a) I would omit the scare quotes (that is an overused and under-meaningful typographical tactic, IMO), and (b) we should stick to simply “OSG” as Miron has requested.

Reply:
fine by me